src/gui/sysConf.cpp

```diff
@@ -882,7 +882,7 @@ bool FurnaceGUI::drawSysConf(int chan, DivSystem type, DivConfig& flags, bool mo
         speakerType=1;
         altered=true;
       }
-      if (ImGui::RadioButton("Piezo",speakerType==2)) {
+      if (ImGui::RadioButton("Moving iron",speakerType==2)) {
```

Comment:
Revert to Piezo. Even if Moving Iron makes more sense, Piezo is what we often say.